catalog/dags/common/urls.py

```diff
@@ -33,6 +37,9 @@ def validate_url_string(url_string, strip_slash: bool = True):
     strip_slash: Flag (bool) decides whether to strip slashes in URL or not,
     Default value is `True`
     """
+    if url_string and " " in url_string:
```

Comment:
Maybe a nit, but I would move this check below L45 so that it is after the log statement about beginning URL validation, and after we have already checked that `url_string` is a non-null string.

You can also remove the unnecessary `else` on [L46](https://github.com/WordPress/openverse/pull/3198/files#diff-18a4035f56c5e8dbe16c08fb824fc99a5479d5974726f159017feb25779df742R46).